tox.ini

```diff
@@ -4,9 +4,10 @@
 # Newton: Swift 2.10.1
 # Ocata: Swift 2.13.1
 # Pike: Swift 2.15.1
+# Queens: Swift 2.17.0
 
 [tox]
-envlist = py27-swift{1.13.1,2.1.0,2.2.0,2.3.0,2.5.0,2.7.1,2.10.1,2.13.1,2.15.1}
+envlist = py27-swift{1.13.1,2.1.0,2.2.0,2.3.0,2.5.0,2.7.1,2.10.1,2.13.1,2.15.1,2.16.0,2.17.0}
```

Comment:
Which OpenStack release brought Swift 2.16.0? Should be added to the list above.

Reply:
2.16.0 is Queens as well as 2.17.0, Fixed